package.json

```diff
@@ -66,6 +66,7 @@
     "@types/react-fontawesome": "^1.6.5",
     "@types/react-modal": "^3.16.0",
     "@types/styled-components": "^5.1.26",
+    "@types/uuid": "^9.0.8",
```

Comment:
can we install @types/uuid as a dev dependency please rather than a dependency? I know the interns did it this way but I don't think this is right. You'll probs have to uninstall @types/uuid and then do `npm install -D @types/uuid`

(unless this leads to next.js being unable to build, in which case find as a dependency)